src/Snakemake/rules/QC/picard.smk

```diff
@@ -0,0 +1,60 @@
+localrules:
+    touchBatch,
+
+
+rule picardHsMetrics:
+    input:
+        bam="DNA_bam/{sample}-ready.bam",
+        intervals=config["bed"]["intervals"],
+    output:
+        "qc/{sample}/{sample}.HsMetrics.txt",
+    log:
+        "logs/qc/picard/HsMetrics/{sample}.log",
+    singularity:
+        config["singularity"]["picard"]
+    shell:
+        "(java -Xmx4g -jar /opt/conda/share/picard-2.20.1-0/picard.jar CollectHsMetrics BAIT_INTERVALS={input.intervals} TARGET_INTERVALS={input.intervals} INPUT={input.bam} OUTPUT={output}) &> {log}"
```

Comment:
[Wrapper 0.72.0/bio/picard/collecthsmetrics](https://snakemake-wrappers.readthedocs.io/en/0.72.0/wrappers/picard/collecthsmetrics.html) ?

Reply:
Wrapper needs more input files than we use at the moment.

Reply:
Why are we not using reference?

Reply:
Don't know. Needs to be investigated.

Reply:
Submitted as an issue (enhancement).

Reply:
#69